dev-lua/luafilesystem/luafilesystem-1.7.0.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit toolchain-funcs
+
+MY_PV=${PV//./_}
+
+DESCRIPTION="File System Library for the Lua Programming Language"
+HOMEPAGE="https://keplerproject.github.io/luafilesystem/"
+SRC_URI="https://github.com/keplerproject/luafilesystem/archive/v${MY_PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~mips ~ppc ~ppc64 ~x86 ~x86-fbsd"
+IUSE="luajit"
+
+RDEPEND="
+	!luajit? ( >=dev-lang/lua-5.1 )
+	luajit? ( dev-lang/luajit:* )"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+PATCHES="${FILESDIR}/${P}.patch"
+
+HTML_DOCS=( doc/us )
+
+S="${WORKDIR}/${PN}-${MY_PV}"
+
+src_prepare() {
+	default
+	eapply_user
+}
+
+src_compile() {
+	emake \
+		CC=$(tc-getCC) \
```

Comment:
`tc-export CC`? :)

Reply:
The current versions of the ebuilds have that in place.  I've made a few changes only to apply the patch and the version bump.